yarn/.ssh/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Can we generate the id_rsa and authorized keys in the Makefile?  They should be different between environments.